llvm/lib/IR/RepoTicket.cpp

```diff
@@ -116,7 +116,37 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
   return accumulateGOInitialDigestAndGetGODigestState(GO, GOHash, GOIMap);
 }
 
-// Update the GO's hash value by adding the hash of its dependents.
+static void updateDigestUseContributions(const GlobalObject *GO, MD5 &GOHash,
+                                         const GOVec &Contributions,
+                                         GOInfoMap &GOIMap) {
```

Comment:
This function (unlike updateDigestUseDependencies() below) doesn't return 'Changed'. Is there a reason for this?

Reply:
I have updated the function and moved 'Changed' inside the function. Thanks.